server/src/main/java/com/cloud/configuration/Config.java

```diff
@@ -1457,9 +1457,8 @@ public enum Config {
     DefaultMaxDomainPrimaryStorage("Domain Defaults", ManagementServer.class, Long.class, "max.domain.primary.storage", "400", "The default maximum primary storage space (in GiB) that can be used for a domain", null),
     DefaultMaxDomainSecondaryStorage("Domain Defaults", ManagementServer.class, Long.class, "max.domain.secondary.storage", "800", "The default maximum secondary storage space (in GiB) that can be used for a domain", null),
     DefaultMaxDomainProjects("Domain Defaults",ManagementServer.class,Long.class,"max.domain.projects","50","The default maximum number of projects that can be created for a domain",null),
-    // Move to configkey?
-    DefaultMaxDomainBuckets("Domain Defaults", ManagementServer.class, Long.class, "max.domain.buckets", "-1", "The default maximum number of buckets that can be created for a domain", null),
-    DefaultMaxDomainObjectStorage("Domain Defaults", ManagementServer.class, Long.class, "max.domain.object.storage", "-1", "The default maximum object storage space (in GiB) that can be used for a domain", null),
+    DefaultMaxDomainBuckets("Domain Defaults", ManagementServer.class, Long.class, "max.domain.buckets", "40", "The default maximum number of buckets that can be created for a domain", null),
+    DefaultMaxDomainObjectStorage("Domain Defaults", ManagementServer.class, Long.class, "max.domain.object.storage", "800", "The default maximum object storage space (in GiB) that can be used for a domain", null),
```

Comment:
same here

Reply:
Moved the new settings to respective service classes.